examples/hpc/dla-driver/src/layers.rs

```diff
@@ -179,6 +179,50 @@ pub fn conv2d_bias_relu<T: DlaOutput + Clone>(
     )
 }
 
+pub fn grouped_conv2d<T: DlaOutput + Clone>(
```

Comment:
Public API function: consider documenting what the function is trying to achieve. Possibly even `# Arguments` as well.